@plotly/prettier-config-dash/README.md

```diff
@@ -0,0 +1,3 @@
+# @plotly/prettier-config-dash
+
+Shareable Prettier configuration for Dash projects.
```

Comment:
Not a big deal, but is it important that this is a separate package? Feels like it would be lighter to use if this were in the same package as the eslint config, and they're intended to be used in tandem.

Reply:
The prettier configuration could be added to the `eslint-...` one but it would need to be the default module export (so including "@plotly/eslint-config-dash" would be invalid and would return the Prettier configuration). It couldn't be the other around because eslint configs must be published under `@<scope>/eslint-config-<package>` or `eslint-config-<package>`

Reply:
OK. I was hoping we could do something like `"prettier": "@plotly/eslint-config-dash/prettier"` but even if that works it does look a bit odd to have prettier explicitly nested under eslint.